src/keybindings.h

```diff
@@ -185,6 +185,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_GOTO_TAGDEFINITION,				/**< Keybinding. */
 	GEANY_KEYS_SEARCH_NEXTMESSAGE,				/**< Keybinding. */
 	GEANY_KEYS_EDITOR_DELETELINETOEND,			/**< Keybinding. */
+	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
```

Comment:
Wait.  This enum is public, one must not add anything in the middle but only at the end.  Right not it changes every following values, breaking API.